src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random RANDOM = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        String color = colorSupplier.getRandomColor().name();
+        int number = new Random().nextInt(101);

```

Comment:
There is no need to create a new instance of Random every time the method is called. You can use the existing 'RANDOM' instance defined at the class level.